apps/src/music/soundsPanel.module.scss

```diff
@@ -0,0 +1,81 @@
+@import "color.scss";
+@import "./theme/tokens-dark";
+
+.soundsPanel {
+  user-select: none;
+}
+
+.folder {
+  background-color: $gray-900;
+  overflow: hidden;
+  border-radius: 4px;
+  padding: 6px;
+  &:first-child {
+    margin-top: 0;
+  }
+  &:not(:first-child) {
+    margin-top: 6px;
+  }
```

Comment:
nit/question: could you just move the `margin-top: 6px` into the `.folder` class and let it get overridden by the `&:first-child` rule? i'm fine with this if it's safer/clearer

Reply:
Ah, I had that first and thought I'd be more pure this way. :)  I think this is okay, but will keep an eye out for compatibility issues.